index.js

```diff
@@ -121,6 +152,26 @@ function handleArguments(env) {
     exit(1);
   }
 
+  customizeConfig(['home', 'project', 'cwd'].map(getConfigFile));
```

Comment:
I think it's better to have the `sortBy` or `map` call be assigned to a variable and to make `customizeConfig` non-mutating with its return value assigned to another variable.  Naming variables for each step usually clarifies the code a lot.